### ISSUE TITLE
Error ensemble_scheduling

### ISSUE THREAD

**Issue by akurniawan** [NVIDIA/tensorrt-inference-server#257]
Hi, I encounter the following error while running the inference server with ensemble platform
```
[libprotobuf ERROR external/protobuf_archive/src/google/protobuf/text_format.cc:307] Error parsing text-format nvidia.inferenceserver.ModelConfig: 23:21: Message type "nvidia.inferenceserver.ModelConfig" has no field named "ensemble_scheduling".
E0429 03:18:53.407096 1 server.cc:620] Can't parse /models/resnet_ensemble/config.pbtxt as text proto
```

The following is the structure of my model repository
<img width="211" alt="image" src="https://user-images.githubusercontent.com/4723643/56875101-ad691980-6a68-11e9-9c2b-b0193703ea2e.png">

And this is my configuration
```
name: "resnet_ensemble"
platform: "ensemble"
max_batch_size: 128
input [
  {
    name: "INPUT"
    data_type: TYPE_STRING
    dims: [ 1 ]
  }
]
output [
  {
    name: "OUTPUT"
    data_type: TYPE_FP32
    dims: [ 1000 ]
  },
  {
    name: "OUTPUT2"
    data_type: TYPE_FP32
    dims: [ 5 ]
  }
]
ensemble_scheduling {
  step [
    {
      model_name: "resnet50_netdef"
      model_version: -1
      input_map {
        key: "0"
        value: "INPUT"
      }
      output_map {
        key: "502"
        value: "OUTPUT"
      }
    },
    {
      model_name: "resnet50_netdef"
      model_version: -1
      input_map {
        key: "0"
        value: "INPUT"
      }
      output_map {
        key: "502"
        value: "OUTPUT2"
      }
    }
  ]
}
```
I use `nvcr.io/nvidia/tensorrtserver:19.02-py3` to run the inference server.

Is there anything that I do wrong? I'm not pretty sure with the structure of the `ensemble` repository though

**Comment by bezero** [comment]
@akurniawan I am not sure about the config file, but you are certainly using the wrong release **(19.02)** for ensemble. As far as I know, it is available starting from **19.04 (currently latest)**  release and it is in beta stage now. You can try switching to ```nvcr.io/nvidia/tensorrtserver:19.04-py3``` which might solve you issue.

**Comment by akurniawan** [comment]
@bezero Ah I see, I couldn't find any [release notes](https://docs.nvidia.com/deeplearning/sdk/inference-release-notes/rel_19-04.html#rel_19-04) saying the ensemble is enabled on 19.04, so I was just assuming it already available on the previous version. Let me try that first. Thanks!

**Comment by deadeyegoodwin** [comment]
@akurniawan parts of ensembling are in 19.04 but the implementation is not complete (which is why it wasn't mentioned in the release notes). Ensembling is available on the master branch and the master branch documentation has ensembling information: 

https://docs.nvidia.com/deeplearning/sdk/tensorrt-inference-server-master-branch-guide/docs/models_and_schedulers.html#ensemble-models

https://docs.nvidia.com/deeplearning/sdk/tensorrt-inference-server-master-branch-guide/docs/client.html#ensemble-image-classification-example-application

Ensembling will be included in the 19.05 release (including being mentioned in the release notes ;-)

Try you model using the master branch and see if it works there...
